SimG4CMS/PPS/interface/PPSVDetectorOrganization.h

```diff
@@ -0,0 +1,18 @@
+///////////////////////////////////////////////////////////////////////////////
+// File: PPSVDetectorOrganization.h
+// Description: Base class for numbering scheme of PPS
+///////////////////////////////////////////////////////////////////////////////
+#ifndef _PPS_VDetectorOrganization_h
+#define _PPS_VDetectorOrganization_h
+
+#include "G4Step.hh"
+#include <boost/cstdint.hpp>
```

Comment:
@mundim , this boost header may be removed - there is #include <cstdint> in the base class header.

Reply:
Done.